server/src/Korga.Server/EmailRelay/DistributionListService.cs

```diff
@@ -11,55 +11,58 @@ namespace Korga.Server.EmailRelay;
 
 public class DistributionListService
 {
-	private readonly DatabaseContext database;
+    private readonly DatabaseContext database;
 
-	public DistributionListService(DatabaseContext database)
-	{
-		this.database = database;
-	}
+    public DistributionListService(DatabaseContext database)
+    {
+        this.database = database;
+    }
 
-	public async ValueTask<EmailRecipient[]> GetRecipients(DistributionList distributionList, long emailId, CancellationToken cancellationToken)
-	{
-		return (await GetPeople(distributionList, cancellationToken))
-		// Avoid duplicate emails for married couples with a shared email address
-			.GroupBy(p => p.Email)
-			.Select(grouping => new EmailRecipient(
-				emailAddress: grouping.Key,
-				fullName: string.Join(", ", grouping.Select(r => r.FirstName)) + ' ' + grouping.First().LastName)
-			{ EmailId = emailId })
-			.ToArray();
-	}
+    public async ValueTask<string[]> GetRecipients(DistributionList distributionList, CancellationToken cancellationToken)
+    {
+        return (await GetPeople(distributionList, cancellationToken))
+            .Select(p  => p.Email)
+            .Distinct()
+            .ToArray();
+        // Avoid duplicate emails for married couples with a shared email address
```

Comment:
Is out commented code necessary? If not, why not remove it.

Reply:
Good point. Strictly speaking this code is not necessary for resending emails, however it will be necessary for forwarding them. I changed the API to reintegrate this code.